config/locales/devise.es.yml

```diff
@@ -0,0 +1,61 @@
+# Traducciones adicionales en https://github.com/plataformatec/devise/wiki/I18n
```

Comment:
:no_good: Uh-oh!  This file has some corrupted encodings in it.  Even if it works programmatically, maintaining this would be painful.  This definitely needs to be fixed.